### PR TITLE
UI/UX changes, RAIN-2942: inbox Card Alignment , RAIN-3174 : Search Param Alignment, RAIN : 3155 : non business service specific status removed, 

### PR DESCRIPTION
**Pull request by harit-egov**

### CODE REVIEW

File: packages/modules/pt/src/components/inbox/search.js [hunk 1]
```diff
@@ -142,18 +143,26 @@ const SearchApplication = ({ onSearch, type, onClose, searchFields, searchParams
                   </div>
                 ))}
               {type === "desktop" && !mobileView && !isInboxPage && (
-                <div style={{ display: "flex", flexDirection: "column", justifyContent: "center", alignItems: "center", marginLeft: "8px" }}>
+                <div
+                  style={{
```
Comment (saurabh-egov) [inline packages/modules/pt/src/components/inbox/search.js:147]: use css

File: packages/modules/pt/src/components/inbox/search.js [hunk 2]
```diff
@@ -142,18 +143,26 @@ const SearchApplication = ({ onSearch, type, onClose, searchFields, searchParams
...
+                    alignItems: "center",
+                    marginLeft: "50%",
+                    maxWidth: "50%",
+                  }}
+                >
                   <SubmitBar
                     className="submit-bar-search"
                     label={t("ES_COMMON_SEARCH")}
-                    style={{ textAlign: "center", marginLeft: "unset", maxWidth: "100%", width: "100%", marginBottom: "20px" }}
+                    style={{ textAlign: "center", marginLeft: "unset", maxWidth: "100%", width: "100%", marginBottom: "20px", marginTop: "unset" }}
```
Comment (saurabh-egov) [inline packages/modules/pt/src/components/inbox/search.js:160]: use css

File: packages/modules/pt/src/pages/citizen/MyProperties/propertyOwnerHistory.js [hunk 3]
```diff
@@ -82,6 +82,86 @@ const propertyOwnerHistory = ({ userType, propertyId: propertyIdFromProp }) => {
...
+        <Card>
+          <div>
+            {Object.keys(ownershipInfo).map((key, index, arr) => {
+              const date = convertEpochToDate(Number(key));
+              return (
+                <div style={{ margin: "0 10px", position: "relative", paddingBottom: "5px" }}>
+                  <div style={{ display: "flex" }}>
+                    <div
+                      style={{
+                        backgroundColor: "#fe7a51",
```
Comment (saurabh-egov) [inline packages/modules/pt/src/pages/citizen/MyProperties/propertyOwnerHistory.js:98]: use css

File: packages/modules/pt/src/pages/citizen/MyProperties/propertyOwnerHistory.js [hunk 4]
```diff
@@ -130,7 +210,13 @@ const propertyOwnerHistory = ({ userType, propertyId: propertyIdFromProp }) => {
                 >
                   &nbsp;-&nbsp;{date}
                 </CardSubHeader>
-                <TransferDetails data={ownershipInfo[key]} />
+                <TransferDetails data={ownershipInfo[key]} wrapperStyles={{
+                  borderLeft: "2px solid rgba(0, 0, 0, 0.12)",
```
Comment (saurabh-egov) [inline packages/modules/pt/src/pages/citizen/MyProperties/propertyOwnerHistory.js:214]: use css

File: packages/modules/pt/src/pages/citizen/MyProperties/propertyOwnerHistory.js (hunk 3 above)
Comment (harit-egov) [reply packages/modules/pt/src/pages/citizen/MyProperties/propertyOwnerHistory.js:98]: tasked Ayan to do it later...

File: packages/modules/pt/src/pages/citizen/MyProperties/propertyOwnerHistory.js (hunk 4 above)
Comment (harit-egov) [reply packages/modules/pt/src/pages/citizen/MyProperties/propertyOwnerHistory.js:214]: asked Ayan to do it later...

File: packages/modules/pt/src/components/inbox/search.js (hunk 1 above)
Comment (harit-egov) [reply packages/modules/pt/src/components/inbox/search.js:147]: corrected

File: packages/modules/pt/src/components/inbox/search.js (hunk 2 above)
Comment (harit-egov) [reply packages/modules/pt/src/components/inbox/search.js:160]: corrected

**Review by jagankumar-egov — APPROVED** [review]

**Pull request merged**